var/spack/repos/builtin/packages/hiop/package.py

```diff
@@ -134,9 +138,10 @@ class Hiop(CMakePackage, CudaPackage, ROCmPackage):
 
     # RAJA > 0.14 and Umpire > 6.0 require c++ std 14
     # We are working on supporting newer Umpire/RAJA versions
-    depends_on("raja@0.14.0:0.14", when="@0.5.0:+raja")
-    depends_on("umpire@6.0.0:6", when="@0.5.0:+raja")
-    depends_on("camp@0.2.3:0.2", when="@0.5.0:+raja")
+    depends_on("raja@0.14.0:0.14", when="@0.3.99:+raja")
+    depends_on("umpire@6.0.0:6", when="@0.3.99:+raja")
+    depends_on("camp@0.2.3:0.2", when="@0.3.99:+raja")
```

Comment:
```suggestion
    depends_on("raja@0.14.0:0.14", when="@0.5.0:+raja")
    depends_on("umpire@6.0.0:6", when="@0.5.0:+raja")
    depends_on("camp@0.2.3:0.2", when="@0.5.0:+raja")
    depends_on("raja@:0.14", when="@0.3.99:+raja")
    depends_on("umpire@:6", when="@0.3.99:+raja")
    depends_on("camp@:0.2", when="@0.3.99:+raja")
```

(I don't think that this syntax is correct, but I am trying to communicate what I am putting in english below)

`hiop@0.3.99` requires RAJA and uses C++11 as the C++ standard. HiOp introduced C++14 as the standard, and so after that release, we are compatible with `raja@0.14.0:0.14`. RAJA 0.14.0 introduced C++14 as a required standard for consuming projects, hence this also required a change in HiOp.

Reply:
Addressed in 11df808814. Take another look to see if I understood you correctly.